### ISSUE TITLE
the model inference 

### ISSUE THREAD

**Issue by zongtianhu** [PointCloudYC/ResPointNet2#5]
Hello, does this repository have code for model inference? I only seem to find training and validation.

**Comment by PointCloudYC** [comment]
No, but with a simple trick, you can do the inference. Just prepare the inference data as the validation data was prepared and assume the datat all belong to one class.

**Comment by PointCloudYC** [comment]
No, but with a simple trick, you can do the inference. Just prepare the inference data as the validation data was prepared and assume the datat all belong to one class.
